src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -31,13 +31,19 @@ class DebugCommand extends Command
 
     private $twig;
     private $projectDir;
+    private $bundlesMetadata;
+    private $twigDefaultPath;
+    private $rootDir;
 
-    public function __construct(Environment $twig, string $projectDir = null)
+    public function __construct(Environment $twig, string $projectDir = null, array $bundlesMetadata = array(), string $twigDefaultPath, string $rootDir)
```

Comment:
New parameters must have a default value to prevent a BC.

Reply:
Sorry for forgetting. I wrote it in the first version and forget it in the second one. Thank you